src/main/java/com/vesoft/nebula/jdbc/statement/NebulaStatementImpl.java

```diff
@@ -19,7 +19,7 @@
 public class NebulaStatementImpl implements NebulaStatement {
     protected final Logger log = LoggerFactory.getLogger(this.getClass());
     protected final String[] updateKeyword = {"update", "delete", "insert", "upsert", "create", "drop", "alter", "rebuild"};
-    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph"};
+    protected final String[] queryKeyword = {"match", "lookup", "go", "fetch", "find", "subgraph", "show"};
```

Comment:
Could you please also add `desc` and `describe`

Reply:
could you plz help add the two keyword above and resolve the conflict?